bundle/manifests/elasticsearch-operator-alerts_monitoring.coreos.com_v1_prometheusrule.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: elasticsearch-operator
+  name: elasticsearch-operator-alerts
+spec:
+  groups:
+  - name: elasticsearch_operator.alerts
+    rules:
+    - alert: ElasticsearchOperatorRemovalWarning
+      annotations:
+        message: Elasticsearch Operator deprecated and will be removed from the catalog
+          on November 2025.
+        summary: In OCP 4.16 and forward, Elasticsearch is only supported for ServiceMesh,
+          Tracing and Kiali, and this operator will be removed from the OpenShift
+          Operator Catalog in November 2025. Elasticsearch is no longer supported
+          for log storage. Kibana is no longer supported in OCP 4.16 and forward.
+      expr: |
+        cluster_version{version=~"4.1(6|7|8).*", type="current"} and on(prometheus) csv_succeeded{exported_namespace="openshift-operators-redhat", name=~"elasticsearch-operator.*", version=~"5.8.*"} > 0
```

(suggestion) last_over_time() would prevent alert resets when one of the metrics is marked as stale. We could also have cluster version and operator version as labels and strip out unnecessary labels. I can help build this.
```suggestion
        last_over_time((cluster_version{version=~"4.1(6|7|8).*", type="current"} and on(prometheus) csv_succeeded{exported_namespace="openshift-operators-redhat", name=~"elasticsearch-operator.*", version=~"5.8.*"})[5m:]) > 0
```

---

@simonpasquier yes please, any help welcome. I could reuse the labels in the summary then.